include/fastdds/dds/topic/DataWriter.hpp

```diff
@@ -20,8 +20,11 @@
 #define _FASTRTPS_DATAWRITER_HPP_
 
 #include <fastdds/rtps/common/Time_t.h>
+#include <fastdds/rtps/common/Guid.h>
+#include <fastdds/rtps/common/InstanceHandle.h>
 #include <fastrtps/qos/DeadlineMissedStatus.h>
-#include <fastrtps/qos/IncompatibleQosStatus.hpp>
+#include <fastdds/dds/qos/IncompatibleQosStatus.hpp>
+#include <fastdds/dds/qos/BaseStatus.h>
```

Comment:
This needed inclusion implies that the forwarded `LivelinessLostStatus` should be removed.

Reply:
Removed